gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageConfiguration.java

```diff
@@ -103,6 +103,7 @@ public Builder blockSize(int value) {
       return this;
     }
 
+
     /** Creates a new instance, but does not destroy the builder. */
     public CloudStorageConfiguration build() {
```

Comment:
I don't understand why we don't like one-line comments, but broke it down to two lines for you (and elsewhere in this file).